x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/GetStackTracesRequest.java

```diff
@@ -239,41 +263,31 @@ public Task createTask(long id, String type, String action, TaskId parentTaskId,
             public String getDescription() {
                 // generating description lazily since the query could be large
                 StringBuilder sb = new StringBuilder();
-                if (indices == null) {
-                    sb.append("indices[]");
-                } else {
-                    sb.append("indices[").append(indices).append("]");
-                }
-                if (stackTraceIds == null) {
-                    sb.append("stackTraceIds[]");
-                } else {
-                    sb.append("stackTraceIds[").append(stackTraceIds).append("]");
-                }
-                if (sampleSize == null) {
-                    sb.append("sample_size[]");
-                } else {
-                    sb.append("sample_size[").append(sampleSize).append("]");
-                }
-                if (requestedDuration == null) {
-                    sb.append(", requested_duration[]");
-                } else {
-                    sb.append(", requested_duration[").append(requestedDuration).append("]");
-                }
-                if (customCostFactor == null) {
-                    sb.append(", custom_cost_factor[]");
-                } else {
-                    sb.append(", custom_cost_factor[").append(customCostFactor).append("]");
-                }
-                if (query == null) {
-                    sb.append(", query[]");
-                } else {
-                    sb.append(", query[").append(Strings.toString(query)).append("]");
-                }
+                appendField(sb, "indices", indices);
+                appendField(sb, "stackTraceIds", stackTraceIds);
```

Comment:
Can we change this to `stacktrace_ids` so it matches the name of the parse field? (doesn't belong into that PR strictly speaking but as you're touching the code it would be nice to make this consistent).

Reply:
Change pushed.

Reply:
Many thanks for fixing this inconsistency that I've introduced! :)